.github/workflows/qa-testbench.yaml

```diff
@@ -0,0 +1,172 @@
+name: QA Testbench run
+
+on:
+  workflow_dispatch:
+    inputs:
+      generation:
+        description: 'Specifies the generation template which should be used by the testbench run'
+        required: false
+        default: 'Zeebe SNAPSHOT'
+      branch:
+        description: 'Specifies the branch, for which the QA Testbench run should be executed'
+        default: 'main'
```

Comment:
:question: I guess this should be stable then?

Reply:
Scheduled workflows only run on the main branch so adding this to the stable/8.0 branch has no effect. If we want to run QA for other branches and versions we'd do this by adding a second schedule on the main branch.